Update h2database from 1.4.200 to 2.1.210 [2]

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Problem Description
<!-- Mandatory -->

Just bumping up the version did not work out in #7978.

### Proposed Change
<!-- Mandatory -->

Update `h2database` and fix all upcoming breaking changes.

### Possible Alternatives
<!-- Optional -->

### Additional Information
<!-- Optional -->

Watch these findings from #7978:
1. h2database now supports:
    - array_to_string function
    - JsonBinaryType

Leftover problems:
- [ ] 1. H2 has problems with columns that are named "year" and "value". Those break the table creation.
<details>
<summary>Table creation with column "year" fails</summary>

The creation of some tables that seems to be connected to their `year` column. There's this error: 

```
Syntax error in SQL statement "create table aggregatereport (id bigint not null, changeDate TIMESTAMP not null, creationDate TIMESTAMP not null, uuid varchar(36) not null, deaths integer, disease varchar(255), epiWeek integer, labConfirmations integer, newCases integer, [*]year integer, district_id bigint, healthFacility_id bigint, pointOfEntry_id bigint, region_id bigint, reportingUser_id bigint, primary key (id))"; expected "identifier"; SQL statement:
```

The [*] points to the place in the statement that is problematic for H2.

</details>
<details>
<summary>CustomizableEnumFacadeEjbTest.createCustomEnums: JdbcSQLSyntaxErrorException: expected "identifier" (value)</summary>

```
javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not prepare statement
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:154)
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1602)
	at org.hibernate.query.Query.getResultList(Query.java:165)
	at org.hibernate.query.criteria.internal.compile.CriteriaQueryTypeQueryAdapter.getResultList(CriteriaQueryTypeQueryAdapter.java:76)
	at de.symeda.sormas.backend.common.BaseAdoService.getAll(BaseAdoService.java:143)
	at de.symeda.sormas.backend.customizableenum.CustomizableEnumValueService$Proxy$_$$_WeldSubclass.getAll(Unknown Source)
	at jdk.internal.reflect.GeneratedMethodAccessor1478.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.jboss.weld.interceptor.proxy.SimpleInterceptionChain.interceptorChainCompleted(SimpleInterceptionChain.java:52)
	at org.jboss.weld.interceptor.chain.AbstractInterceptionChain.invokeNextInterceptor(AbstractInterceptionChain.java:83)
	at org.jboss.weld.interceptor.proxy.InterceptorInvocationContext.proceed(InterceptorInvocationContext.java:146)
	at info.novatec.beantest.transactions.TransactionalInterceptor.manageTransaction(TransactionalInterceptor.java:81)
	at jdk.internal.reflect.GeneratedMethodAccessor46.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.jboss.weld.interceptor.proxy.SimpleMethodInvocation.invoke(SimpleMethodInvocation.java:30)
	at org.jboss.weld.interceptor.chain.AbstractInterceptionChain.invokeNext(AbstractInterceptionChain.java:103)
	at org.jboss.weld.interceptor.chain.AbstractInterceptionChain.invokeNextInterceptor(AbstractInterceptionChain.java:81)
	at org.jboss.weld.interceptor.proxy.InterceptorMethodHandler.executeInterception(InterceptorMethodHandler.java:48)
	at org.jboss.weld.interceptor.proxy.InterceptorMethodHandler.invoke(InterceptorMethodHandler.java:41)
	at org.jboss.weld.bean.proxy.CombinedInterceptorAndDecoratorStackMethodHandler.invoke(CombinedInterceptorAndDecoratorStackMethodHandler.java:53)
	at de.symeda.sormas.backend.customizableenum.CustomizableEnumValueService$Proxy$_$$_WeldSubclass.getAll(Unknown Source)
	at de.symeda.sormas.backend.customizableenum.CustomizableEnumValueService$Proxy$_$$_WeldClientProxy.getAll(Unknown Source)
	at de.symeda.sormas.backend.customizableenum.CustomizableEnumFacadeEjbTest.createCustomEnums(CustomizableEnumFacadeEjbTest.java:23)
...
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1593)
	... 51 more
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "select customizab0_.id as id1_18_, customizab0_.changeDate as changeda2_18_, customizab0_.creationDate as creation3_18_, customizab0_.uuid as uuid4_18_, customizab0_.caption as caption5_18_, customizab0_.dataType as datatype6_18_, customizab0_.description as descript7_18_, customizab0_.descriptionTranslations as descript8_18_, customizab0_.diseases as diseases9_18_, customizab0_.properties as propert10_18_, customizab0_.translations as transla11_18_, customizab0_.[*]value as value12_18_ from CustomizableEnumValue customizab0_ order by customizab0_.changeDate desc"; erwartet "identifier"
Syntax error in SQL statement "select customizab0_.id as id1_18_, customizab0_.changeDate as changeda2_18_, customizab0_.creationDate as creation3_18_, customizab0_.uuid as uuid4_18_, customizab0_.caption as caption5_18_, customizab0_.dataType as datatype6_18_, customizab0_.description as descript7_18_, customizab0_.descriptionTranslations as descript8_18_, customizab0_.diseases as diseases9_18_, customizab0_.properties as propert10_18_, customizab0_.translations as transla11_18_, customizab0_.[*]value as value12_18_ from CustomizableEnumValue customizab0_ order by customizab0_.changeDate desc"; expected "identifier"; SQL statement:
select customizab0_.id as id1_18_, customizab0_.changeDate as changeda2_18_, customizab0_.creationDate as creation3_18_, customizab0_.uuid as uuid4_18_, customizab0_.caption as caption5_18_, customizab0_.dataType as datatype6_18_, customizab0_.description as descript7_18_, customizab0_.descriptionTranslations as descript8_18_, customizab0_.diseases as diseases9_18_, customizab0_.properties as propert10_18_, customizab0_.translations as transla11_18_, customizab0_.value as value12_18_ from CustomizableEnumValue customizab0_ order by customizab0_.changeDate desc [42001-210]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.getSyntaxError(DbException.java:265)
	at org.h2.command.Parser.readIdentifier(Parser.java:5759)
	at org.h2.command.Parser.readTermObjectDot(Parser.java:4882)
	at org.h2.command.Parser.readTerm(Parser.java:5201)
	at org.h2.command.Parser.readFactor(Parser.java:3423)
	at org.h2.command.Parser.readSum(Parser.java:3410)
	at org.h2.command.Parser.readConcat(Parser.java:3375)
	at org.h2.command.Parser.readCondition(Parser.java:3160)
	at org.h2.command.Parser.readExpression(Parser.java:3076)
	at org.h2.command.Parser.parseSelectExpressions(Parser.java:2876)
	at org.h2.command.Parser.parseSelect(Parser.java:2893)
	at org.h2.command.Parser.parseQueryPrimary(Parser.java:2783)
	at org.h2.command.Parser.parseQueryTerm(Parser.java:2650)
	at org.h2.command.Parser.parseQueryExpressionBody(Parser.java:2629)
	at org.h2.command.Parser.parseQueryExpressionBodyAndEndOfQuery(Parser.java:2622)
	at org.h2.command.Parser.parseQueryExpression(Parser.java:2615)
	at org.h2.command.Parser.parseQuery(Parser.java:2576)
	at org.h2.command.Parser.parsePrepared(Parser.java:707)
	at org.h2.command.Parser.parse(Parser.java:674)
	at org.h2.command.Parser.parse(Parser.java:644)
	at org.h2.command.Parser.prepareCommand(Parser.java:551)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:615)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:553)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:149)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:176)
	... 67 more

```

</details>
<details>
<summary>WeeklyReportFacadeEjbTest.testGetByEpiWeekAndUser: JdbcSQLSyntaxErrorException: expected "identifier" (year)</summary>

```
javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not prepare statement
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:154)
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1602)
	at org.hibernate.query.Query.getResultList(Query.java:165)
	at org.hibernate.query.criteria.internal.compile.CriteriaQueryTypeQueryAdapter.getResultList(CriteriaQueryTypeQueryAdapter.java:76)
	at de.symeda.sormas.backend.util.QueryHelper.getSingleResult(QueryHelper.java:228)
	at de.symeda.sormas.backend.util.QueryHelper.getSingleResult(QueryHelper.java:188)
	at de.symeda.sormas.backend.report.WeeklyReportService.getByEpiWeekAndUser(WeeklyReportService.java:113)
	at de.symeda.sormas.backend.report.WeeklyReportService$Proxy$_$$_WeldSubclass.getByEpiWeekAndUser(Unknown 
...
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1593)
	... 78 more
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "select weeklyrepo0_.id as id1_73_, weeklyrepo0_.changeDate as changeda2_73_, weeklyrepo0_.creationDate as creation3_73_, weeklyrepo0_.uuid as uuid4_73_, weeklyrepo0_.assignedOfficer_id as assigne10_73_, weeklyrepo0_.changeDateOfEmbeddedLists as changeda5_73_, weeklyrepo0_.community_id as communi11_73_, weeklyrepo0_.district_id as distric12_73_, weeklyrepo0_.epiWeek as epiweek6_73_, weeklyrepo0_.healthFacility_id as healthf13_73_, weeklyrepo0_.reportDateTime as reportda7_73_, weeklyrepo0_.reportingUser_id as reporti14_73_, weeklyrepo0_.totalNumberOfCases as totalnum8_73_, weeklyrepo0_.[*]year as year9_73_ from weeklyreport weeklyrepo0_ where weeklyrepo0_.epiWeek=7 and weeklyrepo0_.year=2022 and weeklyrepo0_.reportingUser_id=? limit ?"; erwartet "identifier"
Syntax error in SQL statement "select weeklyrepo0_.id as id1_73_, weeklyrepo0_.changeDate as changeda2_73_, weeklyrepo0_.creationDate as creation3_73_, weeklyrepo0_.uuid as uuid4_73_, weeklyrepo0_.assignedOfficer_id as assigne10_73_, weeklyrepo0_.changeDateOfEmbeddedLists as changeda5_73_, weeklyrepo0_.community_id as communi11_73_, weeklyrepo0_.district_id as distric12_73_, weeklyrepo0_.epiWeek as epiweek6_73_, weeklyrepo0_.healthFacility_id as healthf13_73_, weeklyrepo0_.reportDateTime as reportda7_73_, weeklyrepo0_.reportingUser_id as reporti14_73_, weeklyrepo0_.totalNumberOfCases as totalnum8_73_, weeklyrepo0_.[*]year as year9_73_ from weeklyreport weeklyrepo0_ where weeklyrepo0_.epiWeek=7 and weeklyrepo0_.year=2022 and weeklyrepo0_.reportingUser_id=? limit ?"; expected "identifier"; SQL statement:
select weeklyrepo0_.id as id1_73_, weeklyrepo0_.changeDate as changeda2_73_, weeklyrepo0_.creationDate as creation3_73_, weeklyrepo0_.uuid as uuid4_73_, weeklyrepo0_.assignedOfficer_id as assigne10_73_, weeklyrepo0_.changeDateOfEmbeddedLists as changeda5_73_, weeklyrepo0_.community_id as communi11_73_, weeklyrepo0_.district_id as distric12_73_, weeklyrepo0_.epiWeek as epiweek6_73_, weeklyrepo0_.healthFacility_id as healthf13_73_, weeklyrepo0_.reportDateTime as reportda7_73_, weeklyrepo0_.reportingUser_id as reporti14_73_, weeklyrepo0_.totalNumberOfCases as totalnum8_73_, weeklyrepo0_.year as year9_73_ from weeklyreport weeklyrepo0_ where weeklyrepo0_.epiWeek=7 and weeklyrepo0_.year=2022 and weeklyrepo0_.reportingUser_id=? limit ? [42001-210]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
...
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:149)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:176)
	... 94 more
```

</details>

- [ ] 2. In CampaignFormMeta, we are using @Type(type = "org.hibernate.type.TextType") for the campaign form elements and translations; this is transformed by H2 into varchar(2147483647) which then produces an error because H2 can't deal with that value. `CampaignFacadeEjbTest.testGetLastStartedCampaign` is failing.

--- a/sormas-backend/src/test/java/de/symeda/sormas/backend/AbstractBeanTest.java
+++ b/sormas-backend/src/test/java/de/symeda/sormas/backend/AbstractBeanTest.java
@@ -245,8 +245,6 @@ public abstract class AbstractBeanTest extends BaseBeanTest {
 		em.getTransaction().begin();
 		Query nativeQuery = em.createNativeQuery("CREATE ALIAS similarity FOR \"de.symeda.sormas.backend.H2Function.similarity\"");
 		nativeQuery.executeUpdate();
-		nativeQuery = em.createNativeQuery("CREATE ALIAS array_to_string FOR \"de.symeda.sormas.backend.H2Function.array_to_string\"");
-		nativeQuery.executeUpdate();
 		nativeQuery = em.createNativeQuery("CREATE ALIAS date_part FOR \"de.symeda.sormas.backend.H2Function.date_part\"");
 		nativeQuery.executeUpdate();
 		nativeQuery = em.createNativeQuery("CREATE ALIAS epi_week FOR \"de.symeda.sormas.backend.H2Function.epi_week\"");
@@ -258,8 +256,6 @@ public abstract class AbstractBeanTest extends BaseBeanTest {
 		nativeQuery = em.createNativeQuery("CREATE ALIAS set_limit FOR \"de.symeda.sormas.backend.H2Function.set_limit\"");
 		nativeQuery.executeUpdate();
 		nativeQuery = em.createNativeQuery("CREATE ALIAS date FOR \"de.symeda.sormas.backend.H2Function.date\"");
-		nativeQuery.executeUpdate();
-		nativeQuery = em.createNativeQuery("CREATE TYPE \"JSONB\" AS other;");
 		nativeQuery.executeUpdate();
 		em.getTransaction().commit();
 	}

--- a/sormas-backend/src/test/java/de/symeda/sormas/backend/ExtendedH2Dialect.java
+++ b/sormas-backend/src/test/java/de/symeda/sormas/backend/ExtendedH2Dialect.java
@@ -1,15 +1,12 @@
 package de.symeda.sormas.backend;
 
-import java.sql.Types;
-
 import org.hibernate.dialect.H2Dialect;
 import org.hibernate.dialect.function.SQLFunctionTemplate;
 import org.hibernate.dialect.function.StandardSQLFunction;
-
-import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
 import org.hibernate.type.StandardBasicTypes;
 
 public class ExtendedH2Dialect extends H2Dialect {
+
 	public final static String UNACCENT = "unaccent";
 	public final static String ILIKE = "ilike";
 	public final static String WINDOW_FIRST_VALUE_DESC = "window_first_value_desc";
@@ -27,7 +24,6 @@ public class ExtendedH2Dialect extends H2Dialect {
 		registerFunction(ARRAY_TO_STRING, new StandardSQLFunction(ARRAY_TO_STRING));
 		registerFunction(CONCAT_FUNCTION, new StandardSQLFunction("concat"));
 		registerFunction(ARRAY_AGG, new StandardSQLFunction(ARRAY_AGG));
-		registerHibernateType(Types.OTHER, JsonBinaryType.class.getName());
 		// The function unaccent is specific to PostgreSQL
 		// With H2 let's make sure it wont fail by making the function "unaccent" do nothing
 		registerFunction(UNACCENT, new SQLFunctionTemplate(StandardBasicTypes.STRING, "?1"));
@@ -43,5 +39,13 @@ public class ExtendedH2Dialect extends H2Dialect {
 				StandardBasicTypes.LONG,
 				"COUNT(?1) OVER (PARTITION BY ?2 RANGE BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)"));
 		registerFunction(GREATEST, new StandardSQLFunction(GREATEST));
+	}
+
+	/**
+	 * Fixes <em>JdbcSQLSyntaxErrorException: Values of types "BOOLEAN" and "INTEGER" are not comparable</em>.
+	 */
+	@Override
+	public String toBooleanValueString(boolean bool) {
+		return bool ? "TRUE" : "FALSE";
 	}
 }

--- a/sormas-backend/src/test/java/de/symeda/sormas/backend/H2Function.java
+++ b/sormas-backend/src/test/java/de/symeda/sormas/backend/H2Function.java
@@ -1,6 +1,5 @@
 package de.symeda.sormas.backend;
 
-import java.util.Arrays;
 import java.util.Date;
 
 import de.symeda.sormas.api.utils.DateHelper;
@@ -13,10 +12,6 @@ public class H2Function {
 
 	public static float similarity(String a, String b) {
 		return a.equalsIgnoreCase(b) ? 1 : 0;
-	}
-
-	public static String array_to_string(String[] array, String delimiter) {
-		return array != null ? String.join(delimiter, Arrays.asList(array)) : null;
 	}
 
 	public static long date_part(String part, Date date) {

--- a/sormas-backend/src/test/resources/META-INF/persistence.xml
+++ b/sormas-backend/src/test/resources/META-INF/persistence.xml
@@ -92,7 +92,7 @@
 		<properties>
       		<property name="hibernate.dialect" value="de.symeda.sormas.backend.ExtendedH2Dialect"/>
 			<property name="javax.persistence.jdbc.driver" value="org.h2.Driver" />
-			<property name="javax.persistence.jdbc.url" value="jdbc:h2:mem:test;MODE=PostgreSQL;DATABASE_TO_LOWER=TRUE" />
+			<property name="javax.persistence.jdbc.url" value="jdbc:h2:mem:test;MODE=PostgreSQL;DATABASE_TO_LOWER=TRUE;NON_KEYWORDS=YEAR,VALUE" />
 <!--			<property name="javax.persistence.jdbc.driver" value="com.p6spy.engine.spy.P6SpyDriver" />-->
 <!--			<property name="javax.persistence.jdbc.url" value="jdbc:p6spy:h2:mem:test;MODE=PostgreSQL;DATABASE_TO_LOWER=TRUE" />-->
 	    	<property name="javax.persistence.jdbc.user" value="sa" />

--- a/sormas-base/pom.xml
+++ b/sormas-base/pom.xml
@@ -785,7 +785,7 @@
 			<dependency>
 				<groupId>com.h2database</groupId>
 				<artifactId>h2</artifactId>
-				<version>1.4.200</version>
+				<version>2.1.210</version>
 				<scope>test</scope>
 				<!-- In-memory-database for bean-test -->
 				<!-- better compatibility to PostgreSQL than HSQLDB -->

--- a/sormas-ui/src/test/java/de/symeda/sormas/ui/AbstractBeanTest.java
+++ b/sormas-ui/src/test/java/de/symeda/sormas/ui/AbstractBeanTest.java
@@ -40,9 +40,7 @@ import com.opencsv.CSVReader;
 
 import de.symeda.sormas.api.Disease;
 import de.symeda.sormas.api.Language;
-import de.symeda.sormas.api.caze.CaseFacade;
 import de.symeda.sormas.api.caze.caseimport.CaseImportFacade;
-import de.symeda.sormas.api.contact.ContactFacade;
 import de.symeda.sormas.api.event.EventFacade;
 import de.symeda.sormas.api.event.EventParticipantFacade;
 import de.symeda.sormas.api.i18n.I18nProperties;
@@ -105,8 +103,6 @@ public abstract class AbstractBeanTest extends BaseBeanTest {
 		EntityManager em = getEntityManager();
 		em.getTransaction().begin();
 		Query nativeQuery = em.createNativeQuery("CREATE ALIAS similarity FOR \"de.symeda.sormas.ui.H2Function.similarity\"");
-		nativeQuery.executeUpdate();
-		nativeQuery = em.createNativeQuery("CREATE ALIAS array_to_string FOR \"de.symeda.sormas.ui.H2Function.array_to_string\"");
 		nativeQuery.executeUpdate();
 		nativeQuery = em.createNativeQuery("CREATE ALIAS similarity_operator FOR \"de.symeda.sormas.ui.H2Function.similarity_operator\"");
 		nativeQuery.executeUpdate();

--- a/sormas-ui/src/test/java/de/symeda/sormas/ui/ExtendedH2Dialect.java
+++ b/sormas-ui/src/test/java/de/symeda/sormas/ui/ExtendedH2Dialect.java
@@ -15,7 +15,6 @@ public class ExtendedH2Dialect extends H2Dialect {
 	public final static String WINDOW_FIRST_VALUE_DESC = "window_first_value_desc";
 	public final static String WINDOW_COUNT = "window_count";
 
-
 	public ExtendedH2Dialect() {
 		super();
 		// needed because of hibernate bug: https://hibernate.atlassian.net/browse/HHH-11938
@@ -38,5 +37,13 @@ public class ExtendedH2Dialect extends H2Dialect {
 			new SQLFunctionTemplate(
 				StandardBasicTypes.LONG,
 				"COUNT(?1) OVER (PARTITION BY ?2 RANGE BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)"));
+	}
+
+	/**
+	 * Fixes <em>JdbcSQLSyntaxErrorException: Values of types "BOOLEAN" and "INTEGER" are not comparable</em>.
+	 */
+	@Override
+	public String toBooleanValueString(boolean bool) {
+		return bool ? "TRUE" : "FALSE";
 	}
 }

--- a/sormas-ui/src/test/java/de/symeda/sormas/ui/H2Function.java
+++ b/sormas-ui/src/test/java/de/symeda/sormas/ui/H2Function.java
@@ -1,6 +1,5 @@
 package de.symeda.sormas.ui;
 
-import java.util.Arrays;
 import java.util.Date;
 
 import de.symeda.sormas.api.utils.DateHelper;
@@ -16,10 +15,6 @@ public class H2Function {
 
 	public static boolean similarity_operator(String a, String b) {
 		return a.equalsIgnoreCase(b) ? true : false;
-	}
-
-	public static String array_to_string(String[] array, String delimiter) {
-		return array != null ? String.join(delimiter, Arrays.asList(array)) : null;
 	}
 
 	public static double set_limit(Double limit) {

--- a/sormas-ui/src/test/resources/META-INF/persistence.xml
+++ b/sormas-ui/src/test/resources/META-INF/persistence.xml
@@ -92,7 +92,7 @@
 		<properties>
       		<property name="hibernate.dialect" value="de.symeda.sormas.ui.ExtendedH2Dialect"/>
       		<property name="javax.persistence.jdbc.driver" value="org.h2.Driver" />
-			<property name="javax.persistence.jdbc.url" value="jdbc:h2:mem:test;MODE=PostgreSQL;DATABASE_TO_LOWER=TRUE" />
+			<property name="javax.persistence.jdbc.url" value="jdbc:h2:mem:test;MODE=PostgreSQL;DATABASE_TO_LOWER=TRUE;NON_KEYWORDS=YEAR,VALUE" />
 	    	<property name="javax.persistence.jdbc.user" value="sa" />
 	    	<property name="javax.persistence.jdbc.password" value="" />
 

